src/conditionals/programs_with_decision_points/medium/q8/Competition.java

```diff
@@ -0,0 +1,32 @@
+package conditionals.programs_with_decision_points.medium.q8;
+
+/*
+ * Write a program that will ask the user for their placement and print out an "award" based on their placement, 
+ * using the following information: 1st place gets a "Gold Trophy", 2nd place gets a "Silver Trophy", 3rd place 
+ * gets a "Bronze Trophy" and everything below that receives a "Consolation Goodie Bag".
```

Comment:
I would add a statement saying that if the user provides invalid input, they should print `Invalid Input!`